### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AntoineGa**
Clarify Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This extension assumes you have the following binaries on your `PATH`:
    * `draft` (optional)
    * `minikube` (optional)
 
-For `kubectl`, `helm` and `draft` the binaries need not be on the system PATH, provided you tell the extension their locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
+For `kubectl`, `helm` and `draft` the binaries need not to be on the system PATH. You tell to the extension their locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
```
Comment (itowlson) [inline README.md]: In the first sentence, please remove the added 'to' - should read 'the binaries need not be...'.  Or (perhaps better) change it to 'the binaries do not need to be...'

Comment (itowlson) [inline README.md]: In the second sentence, please remove 'to' - should be 'You tell the extension...'  Also, do you think something like "In this case, you tell the extension..." would be clearer still?  Not sure and would value your feedback.

Comment (AntoineGa) [reply README.md]: i suggest "You can configure the extension by specifying the locations"

Comment (AntoineGa) [reply README.md]: Thank you for your kindness

**Pull request merged**